CODEOWNERS

```diff
@@ -62,8 +62,8 @@
 /jbmc/src/jdiff/ @peterschrammel
 /src/cpp/ @kroening @tautschnig @peterschrammel
 /src/solvers/smt2 @kroening @martin-cs @peterschrammel @TGWDB
-/src/solvers/smt2_incremental @peterschrammel @thomasspriggs @TGWDB @esteffin
-/src/solvers/Makefile @kroening @tautschnig @peterschrammel @TGWDB @esteffin
+/src/solvers/smt2_incremental @peterschrammel @thomasspriggs @TGWDB
```

Comment:
I wouldn't expect thomasspriggs is going to contribute anymore.

Reply:
Even if Thomas does: given Thomas lacks write access at this point, code ownership wouldn't entail a sufficient approval anymore.